webextensions/match_patterns.json

```diff
@@ -5,7 +5,7 @@
         "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Match_patterns",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "≤65"
           },
           "edge": {
```

Comment:
@queengooborg any reason for not mirroring edge?

Reply:
Same reasons as described in https://github.com/mdn/browser-compat-data/pull/23451#discussion_r1650280709 -- it's because we have Legacy Edge data, so mirroring would replace that data!